packages/app/src/specs/TestingTypePromo.vue

```diff
@@ -0,0 +1,295 @@
+<template>
+  <div class="flex h-full">
+    <Promo
+      :campaign="campaign"
+      medium="Not set up"
+      :instance-id="promoInstanceId"
+      class="m-auto mt-[32px]"
+    >
```

Comment:
```suggestion
  <div class="flex">
    <Promo
      :campaign="campaign"
      medium="Not set up"
      :instance-id="promoInstanceId"
      class="m-auto"
    >
```

Reply:
I removed the `h-full`, but I think keeping the top margin is more accurate, the design doesn't seem to call for it to be vertically centered in the space, just down a little from the top.